From 889c9dd3948378b69d818996a94519fef79ac59e Mon Sep 17 00:00:00 2001
From: Dmitry Baryshkov <dmitry.baryshkov@linaro.org>
Date: Thu, 13 Jul 2023 23:38:44 +0200
Subject: [PATCH] arm64: dts: qcom: apq8096-db820c: keep s2 regulator
 always on

Needed to keep the GPU alive in our farm setup.

Signed-off-by: Dmitry Baryshkov <dmitry.baryshkov@linaro.org>
Signed-off-by: David Heidelberg <david@ixit.cz>
Upstream-Status: Inappropriate [lame workaround instead of CPR3 driver]
---
 arch/arm64/boot/dts/qcom/apq8096-db820c.dtsi | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm64/boot/dts/qcom/apq8096-db820c.dtsi b/arch/arm64/boot/dts/qcom/apq8096-db820c.dtsi
index 0c076852b494..e3b1c11b8d69 100644
--- a/arch/arm64/boot/dts/qcom/apq8096-db820c.dtsi
+++ b/arch/arm64/boot/dts/qcom/apq8096-db820c.dtsi
@@ -701,6 +701,7 @@ vdd_gfx: s2 {
 		regulator-name = "VDD_GFX";
 		regulator-min-microvolt = <980000>;
 		regulator-max-microvolt = <980000>;
+		regulator-always-on;
 	};
 };
 
